src/test/java/reposense/authorship/FileAnalyzerTest.java

```diff
@@ -15,52 +13,51 @@
 import reposense.model.Author;
 import reposense.model.CommitHash;
 import reposense.template.GitTestTemplate;
-import reposense.util.TestUtil;
 
 
 public class FileAnalyzerTest extends GitTestTemplate {
 
     @Test
     public void blameTest() {
+        config.setSinceDate(BLAME_TEST_SINCE_DATE);
+        config.setUntilDate(BLAME_TEST_UNTIL_DATE);
```

Comment:
if this applies to all blame test, prob should add it to before

Reply:
This does not apply to all blame tests, but majority of them. Should I still add it to `@before` and overwrite accordingly for those tests that do not use the above since and until dates?